### ISSUE TITLE
Reveal in "Explorer View" stopped working with most recent update

### ISSUE THREAD

**Issue by ctodesco** [microsoft/vscode#198120]
Type: <b>Bug</b>

I use shortcut "View: Show Explorer" (default ctrl+shift+E) to focus the Explorer sidebar and highlight/show the currently open file.  There is also another shortcut "File: Reveal Active File in Explorer View" which is supposed to do the same thing.  

Neither of these shortcuts work anymore.  Instead they both open "Open Editors" panel instead.  I NEVER WANT THE "OPEN EDITORS" PANEL!!!  Why does this always open for this shortcut?  I want to see the file in its location in the Explorer so I can see files in the same directory.  

VS Code version: Code 1.84.2 (1a5daa3a0231a0fbba4f14db7ec463cf99d7768e, 2023-11-09T10:51:52.184Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz (8 x 2304)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.61GB (2.40GB free)|
|Process Argv|--crash-reporter-id 539260d6-11b3-42a7-aaa3-15de50524573|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.4.2
vscode-great-icons|emm|2.1.102
vsc-material-theme-icons|equ|3.1.4
file-icons|fil|1.1.0
import-sorter|mik|3.3.1
theme-monokai-pro-vscode|mon|1.2.1
material-icon-theme|PKi|4.31.0
ayu|tea|1.0.5
vscode-icons|vsc|12.6.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263:30880072
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
282f8724:30602487
f6dab269:30613381
a9j8j154:30646983
showlangstatbar:30737416
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxpt:30866567
pythonnoceb:30805159
synctok:30869157
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixtcf:30871695
pythonregdiag2:30871582
pyreplss2:30886141
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
dsvsc016cf:30886111
dsvsc017:30886112
dsvsc018cf:30886115
aa_t_chat:30882232

```

</details>

**Comment by mishazawa** [comment]
Same issue. I've rebind hotkeys to make it work. But "Open Editors" really annoying.

**Comment by lramos15** [comment]
Hmm. I'm unable to repro. Do you have any specific explorer settings?

**Comment by mishazawa** [comment]
I think `explorer.openEditors.visible` key is cause of this issue. Previously, it accepted `0` value, but now it should be `> 0`.

**Comment by gjsjohnmurray** [comment]
@mishazawa I think previously it was the case that setting `"explorer.openEditors.visible": 0` was the way to hide Open Editors view. At some point this was changed. Now you hide it from the context menu of its title, and the minimum permitted value of that setting is 1.

@ctodesco please paste a GIF recording of your steps.

/gifPlease

**Comment by VSCodeTriageBot** [comment]
Thanks for reporting this issue! Unfortunately, it's hard for us to understand what issue you're seeing. Please help us out by providing a screen recording showing exactly what isn't working as expected. While we can work with most standard formats, `.gif` files are preferred as they are displayed inline on GitHub. You may find https://gifcap.dev helpful as a browser-based gif recording tool.

If the issue depends on keyboard input, you can help us by enabling screencast mode for the recording (`Developer: Toggle Screencast Mode` in the command palette). Lastly, please attach this file via the GitHub web interface as emailed responses will strip files out from the issue.

Happy coding!

**Comment by mishazawa** [comment]
@gjsjohnmurray I got it. Is there any way to hide them permanently? When I switch to search pane or git pane and back to explorer, "open editors" appears again and need to be closed manually.

**Comment by mishazawa** [comment]
@lramos15 Is there a way to hide it completely?

**Comment by gjsjohnmurray** [comment]
@mishazawa what exactly seems to trigger its reappearance? And does this also happen after to run the Command Palette command to reload with all extensions disabled?

**Comment by lramos15** [comment]
It does not reappear for me afterwards and I've had it hidden for quite some time. Could you try reloading with all extensions disabled and seeing if maybe an extension is causing something weird

**Comment by mishazawa** [comment]
@gjsjohnmurray 

> what editor tab is active 

It can be either text file or sidebar or terminal.  

> a readonly one

I don't have read-only tabs

I use keyboard shortcut cmd+shift+e to switch to Explorer. This shortcut was removed in new version and I'm setup it manually. I think it is misconfiguration. 

<img width="1122" alt="image" src="https://github.com/microsoft/vscode/assets/7611372/fead5cd3-0171-4559-99ea-535fad58fe56">

Yes. I did this.
